youtube_dl/extractor/tele5.py

```diff
@@ -0,0 +1,40 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from .nexx import NexxIE
+
+
+class Tele5IE(InfoExtractor):
+    _VALID_URL = r'https://www.tele5.de/[mediathek/filme-online/videos|tv/]'
```

Comment:
Incorrect.

Reply:
Read some doc on regular expressions. Namely what does `[]` and `()` mean and how it should be used.